x-pack/legacy/plugins/canvas/public/components/workpad_page/workpad_interactive_page/index.js

```diff
@@ -58,6 +58,19 @@ const configuration = {
   tooltipZ: 1100,
 };
 
+function closest(s) {
```

Comment:
where did this polyfill come from? Might be a good idea to mention its origin. Is this also not something we can just install automatically by adjusting a config somewhere?